schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -42,11 +42,13 @@ func BenchmarkEncoder(b *testing.B) {
 			Schema: studentSchema,
 		},
 	}
+	buf := new(bytes.Buffer)
 	for i := range testCases {
 		tc := testCases[i]
 		b.Run(tc.Name, func(b *testing.B) {
 			for i := 0; i < b.N; i++ {
-				enc := jsonschema.NewEncoder(new(bytes.Buffer))
+				buf.Truncate(0)
```

Comment:
See the docs:

> Truncate discards all but the first n unread bytes from the buffer but continues to use the same allocated storage.

So at least we need to allocate a new buffer per sub-test, even if we don't do so within the loop itself...  That should eliminate the chance of Grow affecting different sub-tests differently.

Or use `ioutil.Discard` as the writer.